### ISSUE TITLE
No worker stats nicehash.com/miner/

### ISSUE THREAD

**Issue by undefy** [nicehash/NiceHashMiner-Archived#1289]
So i noticed i cannot see any of my workers like i used to. Just the overall performance. Why is that? I need to see them individually like before so i can monitor my mining operations for issues...

Any ideas how i can view them?

**Comment by andriansuryadinata2599** [comment]
Same with me, cant see a worker

**Comment by Logic-Elliven** [comment]
Same here.
The choice to choose to see the details of what I am currently mining just isnt in the dropdown menu..!?
![no choice to see details in dropdown](https://user-images.githubusercontent.com/28576299/30958440-71f7ab16-a43d-11e7-97d8-d318aeb1083f.jpg)

**Comment by Logic-Elliven** [comment]
Also note the **payout discrepancy!!!!**

![payout descrepancy small pic adress edited](https://user-images.githubusercontent.com/28576299/30959474-8e7d03b4-a440-11e7-9949-8715ab4fd7dc.jpg)

While we're at it;  Where the hell have 48 Sol/s disappeared to?!

I do hope there isn't some sort of malware 'in the middle' involved here?

If you're wondering why I am running 2 instances of NH, its because NH does NOT prioritise what's most lucrative for each disparate GPU but would rather group them all together mining one or two of the same thing,
My 290x would be mining Equihash with the 3GB 280x atm if one instance was running.
The R9 380 in the right hand group would be making a bit more if it were mining dagger Decred, not Dagger Lbry atm...
ie: The algorithm being used and/or data reaching the server takes priority over profit..!

**Comment by undefy** [comment]
So is this temporary or permanent? I don't really want to switch to a different service but I have over 20 rigs that I need to monitor....

**Comment by BugerDread** [comment]
Same here, the page shows 0 workers, but overall profitability seems to be correct

**Comment by Logic-Elliven** [comment]
My opinion:
Nicehash are a bunch of idiots with high IQs who think their client base consists solely of people too fucking stupid to set up cards to mine directly to pools their buys mine to in the 1st place, and bypass them completely.
That said group far outnumber those capable of creating a batch file (downloadable from said pools), or finding the site whattomine.com
None of us could possibly have a MCSE and simply be so sick and fucking tired of computers that we couldn't be bothered.  
That they can just ignore this until a certain threshold # of people complain.
Well  they are doing a great job of motivating me!
I need ETH for future, post ice age, POS mining anyway, so cheers NiceCunts!
Fuck you very much for the intro to mining and thx for the bin.zip and bin_3rdparty.zip files.

**Comment by BugerDread** [comment]
I dont think that NiceHash stuff are bunch of idiots just because worker stats are temporary not working correctly.

And you simply cant compare pool mining (which you can start using batch file) and NiceHashMiner (which switches algos according to profitability automatically) - these are two different approaches and is up to you which one you choose - I prefer NHM, because for me it makes no sense to mine less profitable coin / algo even if I belive that the less profitable coin will go up shortly because in this case I can buy these "less profitable coins" for bitcoins from nicehash, so I get more of them at the end, but this is not question of mining, but trading...

One thing that can be improved in NHM - profit calculation including electricity price, as I mentioned here: https://github.com/nicehash/NiceHashMiner/issues/1292

Using this let say "netto profit calculation" will choose the most profitable algo more precisely, if you want you can buy any other coin and hold it, but again, this is not question of mining but trading and speculation, mining is about to get as much BTC (USD) as possible in given timeframe which is exactly what NHM does.

**Comment by PhatWheZ** [comment]
Im all for the simplicity in not having to deal with exchanging ETH to BTC,
etc
I personally prefer to get paid in BTC instead of anything else, as in my
country i can trade BTC to my local currency, allowing me to continue to
invest and grow my setup.
For the % cut that NH takes im happy with that arrangement.

Only reason i would not use NH would be if i build a rig that does not work
well on Windows

**Comment by undefy** [comment]
Everything seems to be in order. Any ideas as to what caused it?

**Comment by PhatWheZ** [comment]
My thoughts are that the API calls got bumped somewhere.
The application i use to monitor on my mobile was still working so i knew
it was just a matter of time before the Devs sorted things out
